keccak256/Cargo.toml

```diff
@@ -7,9 +7,9 @@ edition = "2018"
 dev-graph = ["halo2/dev-graph", "plotters"]
 
 [dependencies]
-halo2 = "0.0"
+halo2 = { git = "https://github.com/NoCtrlZ/halo2", rev = "d93c031c127a1d06b228653fafafdb8bb54bb4d2" }
```

Comment:
Shouldn't we try to merge your `halo2` changes into `appliedzkp/halo2` and use it from there?

Also, bear in mind that the patch in the workspace `Cargo.toml` will overwrite this info.

Reply:
This is the PR of this but it depends other PR so it takes time to be merged.
https://github.com/appliedzkp/halo2/pull/8
Should I wait for these PR merged?